apps/researcher/src/app/[locale]/communities/[slug]/actions.ts

```diff
@@ -1,22 +1,35 @@
 'use server';
 
-import {joinCommunity, updateDescription} from '@/lib/community';
+import {joinCommunity, updateCommunity} from '@/lib/community';
 import {revalidatePath} from 'next/cache';
 
 interface UpdateDescriptionAndRevalidateProps {
   communityId: string;
```

Comment:
Or just `id` (since the namespace `updateCommunityAndRevalidate` already makes clear it's about the community)?